### PR TITLE
normalized

### LINKED ISSUE #43: sha fails on windows platform
look at [this](https://github.com/eo-cqrs/cmig/actions/runs/5855496148/job/15873420847?pr=42)

### PR DESCRIPTION
**Pull request by l3r8yJ**
closes #43 

@h1alexbel take a look, please

---

## PR-Codex overview
This PR adds support for Windows and Java 20, updates dependencies, and introduces Unixized library for normalized file content.
### Detailed summary
- Added support for Windows and Java 20 in the Maven workflow
- Updated dependencies: added `unixized` library
- Modified `ShaTest.java`: updated expected hash value
- Modified `Sha.java`: added normalization of file content using `Normalized` class
- Modified `pom.xml`: added `unixized` dependency, updated versions of `hamcrest-all` and `guava`
- Modified `StateChanges.java`: replaced `TextOf` with `UnixizedOf` for file content normalization

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by zoeself** [comment]
@l3r8yJ thank you for your Pull Request. I'll assign someone to review it soon. 

 If this PR solves a ``todo`` from the code, please don't forget to remove it.

**Comment by zoeself** [comment]
@h1alexbel please review this Pull Request. Deadline (when it should be merged or closed) is ``2023-08-18T12:49:06.462400``.

You should check if the requirements have been implemented (partially or in full), if there are unit tests covering the changes and if the CI build passes. Feel free to reject the PR or ask for changes if it's too big or not clear enough.

Estimation here is ``30 minutes``, that's how much you will be paid. You will be paid even if this PR gets rejected.

File: .github/workflows/mvn.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     name: Tests
     strategy:
       matrix:
-        os: [ ubuntu-20.04, macos-12 ]
+        os: [ ubuntu-20.04, macos-12, windows-latest ]
```
Comment (h1alexbel) [inline .github/workflows/mvn.yml]: @l3r8yJ can you specify a fixed version

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: .github/workflows/mvn.yml (hunk 1 above)
Comment (l3r8yJ) [reply .github/workflows/mvn.yml]: @h1alexbel I don't know about windows versions, which one should I set?

**Comment by l3r8yJ** [comment]
@h1alexbel take a look, please

**Review by h1alexbel — APPROVED** [review]
lgtm, @rultor merge

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by l3r8yJ** [comment]
@h1alexbel rultor don't see messages from code review : (

see [this](https://github.com/yegor256/rultor/issues/1779)

**Comment by zoeself** [comment]
@h1alexbel thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).

**Pull request merged**